dns/unbound-plus/src/opnsense/mvc/app/views/OPNsense/Unboundplus/miscellaneous.volt

```diff
@@ -50,5 +99,14 @@
                 });
             });
         });
+
+        $("#saveAct").click(function(){
```

Comment:
we have a simple wrapper for these now, https://github.com/opnsense/core/blob/e330bb46f24e98c4fd81dfa36d754bf7a19b0873/src/opnsense/mvc/app/views/OPNsense/Unbound/miscellaneous.volt#L35-L44

Reply:
@AdSchellevis how would this button look like when using with grid's to reload? Just leave out the form stuff?

Reply:
@mimugmail there’s an onAction handler, like https://github.com/opnsense/core/blob/50f0b15198c87fb1ac1ec4dedaa9518912db91d4/src/opnsense/mvc/app/views/OPNsense/Firewall/alias.volt#L429

I really need to document the JavaScript helpers we have availability sometime, it’s on the list, but time consuming...